poetry/console/commands/add.py

```diff
@@ -50,6 +50,7 @@ class AddCommand(InstallerCommand, InitCommand):
             "Output the operations but do not execute anything (implicitly enables --verbose).",
         ),
         option("lock", None, "Do not perform operations (only update the lockfile)."),
+        option("develop", None, "Add as dependency in editable mode."),
```

Comment:
Please use `--editable`

Reply:
The documentation should also mention that this is only supported for `VCS` and `path` dependnecies.

Reply:
@abn My bad, thought documentation for cli options were automatically generated